test/test-pmctl-local.js

```diff
@@ -40,7 +39,7 @@ helper.test('pmctl', function(t, pm) {
     t.expect(pmctl('restart', '1'), 'Service.*restarting');
     t.waiton(pmctl('get-process-count', '1'), /processes: 2/);
 
-    t.expect(pmctl('env-get', '1', '0'), 'No matching environment variables defined');
+    t.expect(pmctl('env-get', '1', '0'), 'No matching env variables defined');
```

Comment:
The second argument to `t.expect` is part of the assertion, it's not the test message.